scripts/analysis.py

```diff
@@ -356,10 +351,190 @@ def plot_in_out_flux(cursor, facility, influx_bool, title, outputname):
                           title, outputname, init_year)
     else:
         multi_line_plot(waste_dict, timestep,
-                       'Years', 'Mass [kg]',
+                        'Years', 'Mass [kg]',
                         title, outputname, init_year)
 
 
+def total_waste_timeseries(cursor):
+    """Plots a stacked bar chart of the total waste mass vs time
+
+    Parameters
+    ----------
+    cursor: sqlite cursor
+        sqlite cursor
+
+    Returns
+    -------
+    null
+    stacked bar chart of waste mass vs time
+    """
+
+    cur = cursor
+    agent_ids = get_agent_ids(cur, 'sink')
+    resources = cur.execute(exec_string(agent_ids,
+                                        'transactions.receiverId',
+                                        'sum(quantity), senderid, time')
+                            + ' GROUP BY time, senderid').fetchall()
+    init_year, init_month, duration, timestep = get_sim_time_duration(cur)
+    waste_dict = collections.OrderedDict({})
+
+    spec_list = []
+    from_reactor = 0
+    from_fuelfab = 0
+    from_separations = 0
+    from_enrichment = 0
+
+    reactor_timeseries = []
+    separations_timeseries = []
+    enrichment_timeseries = []
+
+    for i in range(0, duration):
+        for row in resources:
+            transaction_time = row[2]
+            if transaction_time == i:
+                senderid = row[1]
+                quantity = row[0]
+                spec = cur.execute("""SELECT spec from
+                                    agententry WHERE
+                                    agentid =""" + str(row[1])).fetchone()
+                if "Reactor" in spec[0]:
+                    from_reactor += quantity
+                elif "Enrichment" in spec[0]:
+                    from_enrichment += quantity
+                elif "Separations" in spec[0]:
+                    from_separations += quantity
+        reactor_timeseries.append(from_reactor/1000)
```

Comment:
This is to convert from kg to MTHM, unless you were pointing out there was a smarter way of doing this?